constants/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "constants"
-version = "0.0.0"
+version = "2.0.0"
 authors = [
```

Comment:
I changed the version to `2.0.0` to test with because otherwise `CheckDriverVersion` would always return incompatible. Obviously, this can't stay like this, so I was wondering how we test with the library without doing this?